src/api/query-hooks/mutations/evidence.tsx

```diff
@@ -12,9 +13,13 @@ import {
  * Update evidence mutation hook used to update evidence in the database
  *
  */
-export function useUpdateEvidenceMutation(param?: {
-  onSuccess?: (evidence: Evidence[]) => void;
-}) {
+export function useUpdateEvidenceMutation(
+  param: {
+    onSuccess?: (evidence: Evidence[]) => void;
+  },
+  incidentId: string
+) {
+  const { refetchIncident } = useIncidentState(incidentId);
```

Comment:
May be a better approach for this is to use: `queryClient.refetchQueries([keys])` to refetch queries, after an update:

```ts
const queryClient = useQueryClient()

queryClient.refetchQueries(keys)
```